pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -455,3 +461,38 @@ func fixLibvirtCrcNetworkActive() (bool, error) {
 	}
 	return true, nil
 }
+
+func checkCrcDnsmasqConfigFile() (bool, error) {
+	c := []byte(crcDnsmasqConfig)
+	_, err := os.Stat(crcDnsmasqConfigPath)
+	if err != nil {
+		return false, fmt.Errorf("File not found: %s: %s", crcDnsmasqConfigPath, err.Error())
+	}
+	config, err := ioutil.ReadFile(crcDnsmasqConfigPath)
+	if err != nil {
+		return false, fmt.Errorf("Error opening file: %s: %s", crcDnsmasqConfigPath, err.Error())
+	}
+	if !bytes.Equal(config, c) {
+		return false, fmt.Errorf("Config file contains changes: %s", crcDnsmasqConfigPath)
+	}
+	return true, nil
+}
+
+func fixCrcDnsmasqConfigFile() (bool, error) {
+	cmd := exec.Command("sudo", "tee", crcDnsmasqConfigPath)
+	cmd.Stdin = strings.NewReader(crcDnsmasqConfig)
+	buf := new(bytes.Buffer)
+	cmd.Stderr = buf
+	err := cmd.Run()
+	if err != nil {
+		return false, fmt.Errorf("Failed to write dnsmasq config file: %s: %s: %v", crcDnsmasqConfigPath, buf.String(), err)
+	}
+
+	sd := systemd.NewHostSystemdCommander()
+	_, esd := sd.Restart("NetworkManager")
+	if esd != nil {
```

Comment:
Can we have `err` only instead `esd`?

Reply:
did this as the err was declared above. changed the structure which prevents the function-scoped use, which I personally thinks makes it less readable, but allows the same name use.